content/en/docs/v3.4/get-started/etcd.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Started with etcd
+weight: 910
+description: Getting started with etcd on Linux or macOS
+---
+
+# Quickstart
+
+This guide shows how to download, setup, and test a simple single-member cluster of etcd.
+
+## Before you start
+
+Before you begin, this etcd guide requires the following:
+
+* Linux or macOS
+
+## Get started
+
+## Step 1 - Download etcd
+
+Set the environment variables to help with the commands that follow.
+
+```
+ETCD_VER=v3.4.15
+
+# choose either URL
+GOOGLE_URL=https://storage.googleapis.com/etcd
+GITHUB_URL=https://github.com/etcd-io/etcd/releases/download
+
+DOWNLOAD_URL=${GITHUB_URL}
```

Comment:
This `DOWNLOAD_URL` is only valid for Linux I believe and so should be moved to the Linux part. The macOS part should use `GOOGLE_URL`.